frontend/src/languages/english.js

```diff
@@ -176,6 +176,7 @@ const en = {
     subproject_currency: "Subproject currency",
     subproject_edit_title: "Edit subproject",
     subproject_permissions_title: "Set permissions for subproject",
+    subproject_permissions_warning: "No permissions allocated for the subproject",
```

Comment:
you can use placeholders directly in the message string, see e.g. `workflow_selection`